yew/src/virtual_dom/vlist.rs

```diff
@@ -1,6 +1,7 @@
 //! This module contains fragments implementation.
 use super::{VDiff, VNode, VText};
 use crate::html::{AnyScope, NodeRef};
+
```

Comment:
It's better not to make unrelated formatting changes in PRs – this is just to make it nicer when someone looks through the git history.